Installation fails on imoprt golint

The package golint has been moved from github.com/golang/lint to golang.org/x/lint, and modules are still not on by default in Go. This means that anyone who doesn't have modules configured gets the following error:
`tehsphinx/exalysis/exam/golint.go:7:2: code in directory [GOPATH]/github.com/golang/lint expects import "golang.org/x/lint".`

--- a/exam/golint.go
+++ b/exam/golint.go
@@ -4,11 +4,11 @@ import (
 	"fmt"
 	"log"
 
-	"github.com/golang/lint"
 	"github.com/logrusorgru/aurora"
 	"github.com/tehsphinx/astrav"
 	"github.com/tehsphinx/exalysis/extypes"
 	"github.com/tehsphinx/exalysis/gtpl"
+	"golang.org/x/lint"
 )
 
 var (

--- a/go.mod
+++ b/go.mod
@@ -2,12 +2,12 @@ module github.com/tehsphinx/exalysis
 
 require (
 	github.com/atotto/clipboard v0.1.0 // indirect
-	github.com/golang/lint v0.0.0-20180702182130-06c8688daad7
 	github.com/kevinburke/go-bindata v3.12.0+incompatible
 	github.com/logrusorgru/aurora v0.0.0-20181002194514-a7b3b318ed4e
 	github.com/pmezard/go-difflib v1.0.0
 	github.com/stretchr/testify v1.2.2
 	github.com/tehsphinx/astrav v0.0.0-20181110120756-b1949977aadc
 	github.com/tehsphinx/clipboard v0.1.3
+	golang.org/x/lint v0.0.0-20181026193005-c67002cb31c3
 	golang.org/x/tools v0.0.0-20180910180008-18207bb12d3a // indirect
 )

